server/bleep/src/semantic.rs

```diff
@@ -199,7 +210,7 @@ impl Semantic {
                     "new chunk",
```

Comment:
Do we still need this `debug` statement?

Reply:
No. Removing it.